datalad/support/json_py.py

```diff
@@ -50,21 +50,28 @@
 from ..dochelpers import exc_str
 
 
-def dump(obj, fname):
+def dump(obj, fname, compressed=False):
```

Comment:
somewhat has mentioned docstrings today ;-) is that why it is still a "Draft"?

Reply:
Yes, there is likely more to come. This PR could grow alongside the demands resulting from the rewrite of metadata aggregation happening in -revolution right now. I made it a draft, because there is no need to merge this piecewise -- and also because I really like this ability to have real draft PRs (watch what someone is working on) instead of the various ways in which we tag real PRs to make them pretend drafts ;-)